### DISCUSSION TITLE
23.11.21 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#74] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 컨트롤러 테스트 작성 (진행 중)
- 내일은 어떤 일을 하나요?
    - 테스트 코드 작성 중 에러 해결

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - PR merge
  - 아티스트 좋아요, 좋아요 조회, 좋아요 취소 기능 구현
  - 테스트 코드 작성 중

- 내일은 어떤 일을 하나요?
  - 테스트 하면서 오류 수정
  - 알림 기능 연결

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 어제 오늘 일을 못했어요
* 내일은 어떤 일을 하나요?
  * 지난번에 한다는 PR 하게 될 것 같습니다.
